test/Tests/Feature/Console/ModelTyperCommandTest.php

```diff
@@ -30,6 +30,9 @@ public function testCommandCanBeExecutedSuccessfully()
     /** @test */
     public function testCommandFailsWhenTryingToResolveAbstractModelThatHasNoBinding()
     {
+        $this->markTestSkipped('Do dont think is needed anymore, since only files that extend Eloquent\Model are considered');
```

Comment:
this test currently fails without it skipping, cause the command itself no longer fails in this situation, so marking as skip for now